api/Window.json

```diff
@@ -3755,7 +3755,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": false
```

Comment:
This looked a bit implausible, but seems to be true: https://software.hixie.ch/utilities/js/live-dom-viewer/saved/8476